trogon/widgets/parameter_controls.py

```diff
@@ -401,26 +381,17 @@ def make_choice_control(
     @staticmethod
     def _make_command_form_control_label(
         name: str | list[str],
-        type: click.ParamType,
+        type: Type[Any],
         is_option: bool,
         is_required: bool,
         multiple: bool,
     ) -> Text:
-        if isinstance(name, str):
-            text = Text.from_markup(
-                f"{name}[dim]{' multiple' if multiple else ''} {type.name}[/] {' [b red]*[/]required' if is_required else ''}"
-            )
-        else:
-            names = Text(" / ", style="dim").join([Text(n) for n in name])
-            text = Text.from_markup(
-                f"{names}[dim]{' multiple' if multiple else ''} {type.name}[/] {' [b red]*[/]required' if is_required else ''}"
-            )
+        names: list[str] = [name] if isinstance(name, str) else name
 
-        if isinstance(type, (click.IntRange, click.FloatRange)):
-            if type.min is not None:
-                text = Text.assemble(text, Text(f"min={type.min} ", "dim"))
-            if type.max is not None:
-                text = Text.assemble(text, Text(f"max={type.max}", "dim"))
+        names = Text(" / ", style="dim").join([Text(n) for n in names])
+        text = Text.from_markup(
+            f"{names}[dim]{' multiple' if multiple else ''} <{type.__name__}>[/] {' [b red]*[/]required' if is_required else ''}"
+        )
```

Comment:
This is much shorter, and roughly equal, but removes respect for click-specific types like `IntRange`.